### PR TITLE
Add check before switching to OAuthToken

### PR DESCRIPTION
**Pull request by dirathea**
Fix issue #105 for Login without params

### CODE REVIEW

**Comment by dirathea** [comment]
I've signed the CLA

**Comment by dirathea** [comment]
I've signed the CLA

File: firebase-auth.html [hunk 1]
```diff
@@ -225,7 +225,8 @@
           case 'google':
           case 'github':
           case 'twitter':
-            if (params.token) {
+          token = (params) ? params.token : undefined;
+            if (token) {
```
Comment (ebidel) [inline firebase-auth.html]: How about just
if (params && params.token)

Comment (dirathea) [inline firebase-auth.html]: Ah I see. so it's less variable.. Alright.. I'll update it

**Comment by robdodson** [comment]
Verified this fixes todo-list app

**Comment by ebidel** [comment]
LGTM

**Pull request merged**